src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -130,4 +133,18 @@ public static void showEmployeeAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredEmployeeList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the project at the given {@code targetIndex} in the
+     * {@code model}'s TaskHub.
+     */
+    public static void showProjectAtIndex(Model model, Index targetIndex) {
```

Comment:
Be warned that there would be an upcoming merge conflict when PR #109 is merged. PR #109 has created almost the same method, but has some additional changes to data fields as well.

Reply:
Noted.